src/content/documentation/publish/third-party-library-usage.md

```diff
@@ -34,7 +37,9 @@ If you don't provide information about third-party libraries and the reviewer ca
 
 ## When must links for third-party libraries be provided?
 
-You must provide a link to the source code for any third-party libraries included in your extension, minified or not.
+Links to third party libraries must be provided when submitting the extension. The links may either be included in the extension's archive, the source archive, or in the "notes for reviewers" on the submission.
+
+If you include links in one of the archives, we recommend either adding them to a README file in the root directory or another root file clearly intended for AMO reviewers.
```

Comment:
While there are many options, in a best practices document let's aim to pick the best practice. The challenge here is that there is no one-size-fits all. For example, if there is no other reason to attach source code, we don't want the developer to do it just for third-party libraries.

In general, notes to reviewers is fine, unless the amount of libraries (and other reviewer information) is too high to be reasonable for a "Notes" field.

Reply:
Makes sense. In that case I think we can simplify this text a bit. WDYT?

```suggestion
Links to third party libraries must be provided when submitting a new version to AMO. You can add the links to the "Notes for Reviewers" section of your extension's details.
```

Reply:
```suggestion
When submitting a version to AMO, links to third-party libraries must be provided. You can add the links to the "Notes for Reviewers" section of your extension's details. 
```

Reply:
Either is fine with me, thanks.